_team/Hongsu Wang.md

```diff
@@ -0,0 +1,6 @@
+---
+name: Hongsu Wang
+designation: MRes student
+photo:/assets//images/team/Hongsu_Wang.jpg
+---
+Hongsu Wang is a MRes student in the Matta group at King's College London.Her project is mainly about molecular dynamics simulations of conjugated polymers.
```

Comment:
Hongsu Wang is a MRes student in the Matta group at King's College London. Her project is performing molecular dynamics simulations of conjugated polymers.